opm/simulators/aquifers/AquiferNumerical.hpp

```diff
@@ -158,6 +156,19 @@ class AquiferNumerical
     // TODO: maybe unordered_map can also do the work to save memory?
     std::vector<int> cell_to_aquifer_cell_idx_;
 
+    inline bool co2store_() const
+    {
+        return ebos_simulator_.vanguard().eclState().runspec().co2Storage();
+    }
+
+    inline bool phaseIdx_() const
```

Comment:
That's a suspicious return type if this is supposed to represent a phase index.

Reply:
Good catch. Its a copy paste from AquiferInterface.hpp. i.e. we have a bug in master. I suppose it works by an accident since the phase indices are 0 and 1. I will fix both here and there.